.github/workflows/pr.yml

```diff
@@ -69,7 +69,9 @@ jobs:
             -Dquarkus.container-image.insecure=true \
             -Dquarkus.helm.repository.push=true \
             -Dquarkus.helm.repository.url=http://localhost:8080/api/charts \
-            -Dquarkus.helm.repository.type=CHARTMUSEUM
+            -Dquarkus.helm.repository.type=CHARTMUSEUM \
+            -Dgithub.repo=${{ github.repository }} \
+            -Dgit.sha.commit=$(git rev-parse --short HEAD)
```

Comment:
in the pull request events, the latest sha commit is in `github.event.pull_request.head.sha` (more info in https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)

Reply:
Fixed: 70e8fd5